spec/models/vote_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Vote do
+  let(:user) { create(:user) }
+  let(:event) { create(:event) }
+  let(:vote) { create(:vote, event: event, user: user) }
+
+  describe 'validation' do
+    it { is.expected_to validate_uniqueness_of(:user).scoped_to(:event) }
+
+    it 'has a valid factory' do
```

Comment:
Probably unnecessary right now but I let it be because it might be useful later

Reply:
In what way could it be useful later? i don't think leaving unneeded code is a good idea. We also have to maintain this code. If we need it in the future, we will add it then :wink: